lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -1605,13 +1605,13 @@ protected function getSQLTableAlias($tableName, $assocName = '')
             $tableName .= '#' . $assocName;
         }
 
-        if (isset($this->currentPersisterContext->sqlTableAliases[$tableName])) {
-            return $this->currentPersisterContext->sqlTableAliases[$tableName];
+        if (isset($this->currentPersisterContext->sqlTableAliases[strtolower($tableName)])) {
+            return $this->currentPersisterContext->sqlTableAliases[strtolower($tableName)];
```

Comment:
This is a map from FQCN to table alias, I'm not sure why `$tableName` is not named `$fqcn` :thinking: